xwalk_android_tests.gypi

```diff
@@ -194,7 +194,6 @@
           ],
         },
       ],
```

Comment:
Yeah, and inclusions are not transitive: the fact that `xwalk_core_shell_apk` includes `java_apk.gypi` only means that it will generate an APK; the fact that this target is a dependency of `xwalk_core_test_apk` only means that it will be built before `xwalk_core_test_apk`, not that it will automatically make it create an APK too.